website/content/en/docs/ansible/reference/advanced_options.md

```diff
@@ -209,3 +209,56 @@ spec:
     served: true
     storage: true
 ```
+
+## Passing Arbitrary Arguments to Ansible
+
+Users can specify an arbitrary command-line argument to the ```operator-sdk exec-entrypoint``` and ```run --local``` subcommands for Ansible-based Operators via the ```ansible-args``` flag.
```

Comment:
we removed the exec-entrypoint https://github.com/operator-framework/operator-sdk/pull/3331. So, it could be made from the new ansible binary that will be provide `ansible-operator run`